pkg/reconciler/eventlistener/eventlistener.go

```diff
@@ -76,6 +76,8 @@ const (
 	defaultConfig = `{"level": "info","development": false,"sampling": {"initial": 100,"thereafter": 100},"outputPaths": ["stdout"],"errorOutputPaths": ["stderr"],"encoding": "json","encoderConfig": {"timeKey": "ts","levelKey": "level","nameKey": "logger","callerKey": "caller","messageKey": "msg","stacktraceKey": "stacktrace","lineEnding": "","levelEncoder": "","timeEncoder": "iso8601","durationEncoder": "","callerEncoder": ""}}`
 
 	triggersMetricsDomain = "tekton.dev/triggers"
+
+	disablePayloadValidationAnnotation = "tekton.dev/disable-payload-validation"
```

Comment:
instead of hardcoding here again can we make use from validation.go file ?
because i see `disablePayloadValidationAnnotation` added in multiple place

Reply:
addressed.